content/en/building/training/training-cards/_index.md

```diff
@@ -1,141 +1,45 @@
 ---
 title: "Training Cards"
-linkTitle: "Training Cards"
-weight: 15
+weight: 2
 description: >
-  Deploy in-app training cards for remote training.
+  Remotely train health workers
 keyword: training
 relatedContent: >
-  building/features/training
-  building/training/training-cards-resources/
+  building/training/training-cards-resources
   building/examples/training
   building/examples/learning-care/
 aliases:
-   - /building/guides/training/training-cards/
-   - /apps/guides/training/training-cards/
+   - /building/features/training/
+   - /apps/features/training/
 ---
-_Introduced in 4.2.0_
```

Comment:
I would add this line on the `building/training/training-cards/` page, as that is the first link the readers might bump into.

Reply:
Done!